src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenReadOnlySpanConstructionTest.cs

```diff
@@ -725,7 +817,7 @@ public void StaticFieldIsUsedForSpanCreatedFromArrayWithInitializer_02()
         {
             // This IL applies CompilerFeatureRequiredAttribute to WellKnownMember.System_ReadOnlySpan_T__ctor_Pointer.
```

Comment:
>This IL applies CompilerFeatureRequiredAttribute to WellKnownMember.System_ReadOnlySpan_T__ctor_Pointer. [](http://example.com/codeflow?start=15&length=104)

Can we add similar test for System_Runtime_CompilerServices_RuntimeHelpers__CreateSpanRuntimeFieldHandle?